raiden-dapp/src/components/ActionButton.vue

```diff
@@ -58,15 +48,18 @@ export default class ActionButton extends Vue {
   @Prop({ type: Boolean, default: false })
   arrow?: boolean;
 
-  @Prop({ type: Boolean, default: false })
-  ghost?: boolean;
-
   @Prop({ type: String, default: '250px' })
   width?: string;
 
   @Prop({ type: Boolean, default: false })
   fullWidth?: boolean;
 
+  @Prop({ type: String, default: '40px' })
+  height?: string;
+
+  @Prop({ type: Boolean, default: false })
+  edgy?: boolean;
```

Comment:
`edgy`? I don't understand this property, name is not obvious.

Reply:
Haha. I thought it is a word. 😂 
I'll change it to `angular`

Reply:
✔️